dataproc_jupyter_plugin/services/dataproc.py

```diff
@@ -20,7 +24,7 @@
 
 
 class Client:
-    def __init__(self, credentials, log, client_session):
+    def __init__(self, credentials, log, dataproc_url, client_session=None):
```

Comment:
This change will break the runtime list functionality since it isn't passing in the dataproc URL.

Did you test that?